_events/2020-09-10-orgfair.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Virtual Org Fair
+featured: null
+location: Online
+address: >-
+  Online
+---
+
+Come join us for the org fair! We will be hosting it online, and we welcome anyone who wants to chat and learn more about KHK. At the virtual org fair, you will have the opportunity to interact with current KHK members and ask any questions you have about our fraternity!
+
+Information provided by the University about this year'
+s org fair:<br/>
```

Comment:
Add another new line before the "information" block and fix the space in "year' s org fair" by removing extra line